src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { taskStatusToCopy } from "constants/task";
+import { TaskStatus } from "types/task";
+
+const { gray, red, yellow, green } = uiColors;
+interface Props {
+  status: TaskStatus;
+  count: number;
+  href?: string;
+  onClick?: () => void;
+}
+export const GroupedTaskStatusBadge: React.FC<Props> = ({
+  count,
+  status,
+  href = "",
+  onClick = () => undefined,
+}) => {
+  let statusDisplayName = taskStatusToCopy[status];
+  if (statusDisplayName.slice(-1) === "s" && count !== 1) {
+    statusDisplayName += "es";
+  } else if (statusDisplayName.slice(-1) === "e" && count !== 1) {
+    statusDisplayName += "s";
+  }
+
+  return (
+    <BadgeContainer
+      css={statusToColorVariant[status]}
+      onClick={onClick}
+      to={href}
```

Comment:
For my original comment i intended for this component to serve as both a clickable button as well as a link. Similar to how the leafygreen button does it https://github.com/mongodb/leafygreen-ui/blob/main/packages/button/src/Button.tsx

Reply:
Cool I will take a look